### PR TITLE
Secure App Insights with managed identity

### PR DESCRIPTION
**Pull request by BigMorty**
## Purpose

* [Secure App Insights with managed identity](https://github.com/Azure-Samples/Azure-Functions-OpenAI-Demo/commit/4d1c9e446474e90fce645e9c30ee98312e6b5133)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: app/frontend/package-lock.json
Comment (paulyuk) [inline app/frontend/package-lock.json:1]: Did this file need to change?  if not i might leave it out of change list.

**Review by paulyuk — COMMENTED** [review]
Please see comment.

**Comment by paulyuk** [comment]
After this PR we should come back and figure out what CI builds are failing.  That's not a good look.

File: app/frontend/package-lock.json
Comment (BigMorty) [reply app/frontend/package-lock.json:1]: There was a library with a med/high issue that 'npm audit fix' updated

**Review by paulyuk — APPROVED** [review]
LGTM thank you

**Pull request merged**